config/storage.yml

```diff
@@ -7,12 +7,12 @@ local:
   root: <%= Rails.root.join("storage") %>
 
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+  secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+  region: us-east-1
+  bucket: your_own_bucket
```

Comment:
my only concern here is this requiring an S3 setup/config step when generating a new app, which we should avoid. Like the gemfile maybe this just becomes a TODO to appear in the config script so people know its here and they have to change it but isn't something that we need to do immediately when booting the new app (or something that will fail silently due to lack of config when someone goes to turn on activestorage). Also, region should be an env var (will vary based on the instance) as should the bucket name

Reply:
That's a good point @hiattp I tried commenting out the gem while still keeping `config.active_storage.service = :amazon` and the app crashes on startup unless all these properties are set. I'll revert the related changes.